### PR TITLE
Add configuration for py.test

### PR DESCRIPTION
**Pull request by wo0dyn**
## Background

I just wanted to setup py.test, and once it was done, I realized some tests were simply not run.
This small PR fixes them and add some configuration in `setup.cfg` for py.test.

## Review

* [x] Tests
* [x] `CHANGELOG.rst` Updated
* [ ] Delete your branch

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,6 +7,7 @@ master (unreleased)
 
 - Added tests for the validators mapping
 - Minor syntax changes
+- Add configuration for py.test
```
Comment (brunobord) [inline CHANGELOG.rst]: you can also add "reactivated accidentally skipped `test_validations.py` tests"

Comment (wo0dyn) [reply CHANGELOG.rst]: Added.

**Comment by wo0dyn** [comment]
> I'd also add a note in the doc mentioning that the tests can be run using py.test (in the "dev" doc?)

Added.

**Comment by wo0dyn** [comment]
It adds lots of file in this PR when I built the documentation. Not sure if every files need to be added here. _Maybe @brunobord knows…_

**Comment by brunobord** [comment]
@wo0dyn a lot of files are just renamed... My opinion is that you should just add the documentation bit and we'll make another branch/PR to inspect what happens when we rebuild the docs.

**Comment by wo0dyn** [comment]
@brunobord Done. Thanks.

**Pull request merged**